canonical-websites/maas.io
# Discourse link navigation

## Done
Add a discourse link to the main navigation. Fix the small screen menu styling

## QA
- Check that the disoucrse link is there
- Check it goes to the right place
- Check small screens look ok

Fixes https://github.com/ubuntudesign/MAAS-squad/issues/831
